php/containers.json

```diff
@@ -585,6 +599,7 @@
       "container_name": "nextcloud-aio-fulltextsearch",
       "display_name": "Fulltextsearch",
       "image": "nextcloud/aio-fulltextsearch",
+      "init": true,
```

Comment:
I wonder do we need this here since fts uses tiny as entrypoint...

Reply:
yes, not needed https://github.com/elastic/dockerfiles/blob/cf5707cefdfc1dba3e95f7d15931aeba5db93f0f/elasticsearch/Dockerfile#L160

Reply:
```suggestion
```